trainings/insights/lesson-plan.md

```diff
@@ -0,0 +1,84 @@
+
+
+# Code Insights Webinar
+
+**Topic:** This training is intended as an initial deep-dive to educate users on Code Insights outside of the 101/102 context. **If at all possible, we recommend having the customer provide a Code Insights use case to you *before* the training, giving you the opportunity to work on that specific insight or something similar;** if not, you can use this as a way to show Code Insights to them and discuss use cases they might find valuable live on the training.
+
+## Intro
+
+Today's training is intended to walk users through our Code Insights product, and show users how to use the feature. By the end of it, we hope that you'll feel comfortable with what Code Insights is and can do, and will be ready to start with your own changes in Sourcegraph.
```

Comment:
```suggestion
Today's training is intended to walk users through our Code Insights product, and show users how to use the feature. By the end of it, we hope that you'll feel comfortable with what Code Insights is and can do, and have created your first insight in Sourcegraph.
```

Can we make the real goal "get them to their first insight"? That would be much more valuable than just leaving them with an understanding. (Also was "changes" left over from a batch changes script? not sure what that referred to).